clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -461,7 +461,7 @@ public enum Command {
   ),
   LIST_CLUSTER_STALE_STORES(
       "list-cluster-stale-stores", "List all stores in a cluster which have stale replicas.", new Arg[] { URL, CLUSTER }
-  ),
+  ), REPUSH_STORE("repush-store", "Repush store", new Arg[] { URL, STORE }),
```

Comment:
Is cluster not a mandatory parameter?

Reply:
It is not.  There's not really any reason why cluster should be a mandatory parameter for most of the admin commands.  Stores are unique to the fabric, and the controller should be smart enough to be able to puzzle the cluster by doing a store discovery step.  So no reason to push that question to the user of the api at this time.

Reply:
Currently, the controller client creation is tied to a cluster. There are validations on the server side where if the request lands on the controller that is not leader for the cluster, it either throws exceptions or potentially gives no response (in some cases)

If the goal is not to have cluster provided as part of the API and have smartness about inferring, that should also go into the controller client API that we expose today which shouldn't require users to pass a cluster.

In the interim, it would rather be cheap to fail fast as opposed to executing the entire request, inferring the cluster of the store (if store related operation) through store metadata and then failing if the handling controller is not a leader for the cluster.